config/core/resources/parallel.yaml

```diff
@@ -187,6 +194,149 @@ spec:
                   uri:
                     type: string
                     description: "the target URI. If ref is provided, this must be relative URI reference."
+          status:
+            type: object
+            x-kubernetes-preserve-unknown-fields: true
+  - name: v1beta1
+    served: true
+    storage: false
+    schema:
+      openAPIV3Schema:
+        type: object
+        properties:
+          spec:
+            required:
+              - branches
+              - channelTemplate
+            type: object
+            properties:
+              branches:
+                type: array
+                description: "the list of filter/subscribers pairs."
+                items:
+                  type: object
+                  required:
+                  - subscriber
+                  properties:
+                    filter:
+                      type: object
+                      description: "the destination of the filter expression that is guarding the branch."
+                      properties:
+                        ref:
+                          type: object
+                          description: "a reference to a Kubernetes object from which to retrieve the target URI."
+                          required:
+                            - apiVersion
+                            - kind
+                            - name
+                          properties:
+                            apiVersion:
+                              type: string
+                              minLength: 1
+                            kind:
+                              type: string
+                              minLength: 1
+                            name:
+                              type: string
```

Comment:
You'll probabaly beed the namespace here as well, or it won't necessarily roundtrip properly. We've had quite a few of these failures because things have drifted over time. Since you need to rebase anyways, maybe just get rid of the schemas. Look for example at inmemorychannel.yaml to see what it looks like without these. You need to add an overall validation section. Plan is to then autogenerate these and merge them rather than code them by hand.

Reply:
ok done. 

Still having an issue with the AdmissionController that cannot decode the new object. This happens just before defaulting. Not sure why yet